string jointOwner
float64 position
Entity meEntity
